Makefile.development

```diff
@@ -0,0 +1,49 @@
+# Default shell
+SHELL := /bin/bash
+
+# Conf vars
+GITHUB_API_KEY ?=
+WWW_PATH ?= /var/www
+LOGS ?= true
+
+#vars
+docker_container_name := doc-srcd-cont
+docker_image_name := quay.io/srcd/docs
+docker_shell := /bin/bash
+caddy_internal_port := 9090
+
+#tools
+docker_run := docker run --rm --detach
+docker_build := docker build -t
+docker_logs := docker logs --follow
+docker_stop := docker stop
+docker_tag := docker tag
+docker_access := docker exec -it
+git_last_commit := $(shell git show --format="%h" --no-patch)
+git_modified = $(shell git status --short)
+
+develop-run: build develop-stop
+	$(docker_build) $(docker_image_name) .;
+	$(docker_run) \
+		--name $(docker_container_name) \
+		--publish $(caddy_internal_port):$(caddy_internal_port) \
+		--env DEBUG_LOG=$(LOGS) \
+		--env GITHUB_API_KEY=$(GITHUB_API_KEY) \
+		$(docker_image_name);
+	$(docker_access) $(docker_container_name) $(docker_shell) || true;
+
+develop-stop:
+	$(docker_stop) $(docker_container_name) || true;
+
```

Comment:
Why you need to return true in case of error? If that's a need of other actions that use this, it should be there, not here

Reply:
It's a way to ensure that if the container does not exist, any other targets that uses this one (`develop-stop`) will succeed anyway.
Otherwise, if the docker container does not exist, the `stop` command would fail and this target would exit with error, so the `develop-run` target (that uses the `develop-stop`) would not success.

Reply:
Yes, I understand that. But that logic should go in those actions not in the `stop` one

Reply:
You're right, fixing it